app/services/audit/replication.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module Audit
+  # Audit the replication of a single PreservedObject.
```

Comment:
... "which may have one or more ZipParts each replicating to multiple configured ZipEndpoints" ?

Now that You are among the chosen few that "get it" ... please try to keep "beginners mind" ... in mind.

Reply:
Agree that this makes clearer. Changing.